ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/qos-profiles.j2

```diff
@@ -66,4 +66,37 @@ qos profile {{ profile.name }}
       shape rate {{ mc_tx_queue.shape.rate }}
 {%         endif %}
 {%     endfor %}
+{%     if profile.priority_flow_control.enabled is arista.avd.defined(true) %}
```

Comment:
An outer `if profile.priority_flow_control is arista.avd.defined` would save us some parsing of all the following sections. You can then add your `!` after that.

Reply:
Agreed